tests/integrations/basic/tests/ui_tests_with_wrapper_script.rs

```diff
@@ -0,0 +1,17 @@
+use std::num::NonZeroUsize;
+use ui_test::*;
+
+fn main() -> ui_test::color_eyre::Result<()> {
+    let config = Config {
+        quiet: false,
+        program: CommandBuilder::cmd("../runner.py"),
```

Comment:
this makes you lose the default arguments (the ones that cause diagnostics to be rendered as json). Ideally just use the default rustc one and overwrite just the `config.program.program` part.